pkg/management/component.go

```diff
@@ -585,17 +585,25 @@ func (c *component) UpdateUser(ctx context.Context, realmName, userID string, us
 	}
 
 	// when the email changes, set the EmailVerified to false
-	if user.Email.Defined && (user.Email.Value == nil || c.isUpdated(user.Email.Value, oldUserKc.Email)) {
-		var verified = false
-		user.EmailVerified = &verified
+	if user.Email.Defined {
+		if user.Email.Value == nil {
+			var verified = false
```

Comment:
Do we still need to remove the flag ?

Reply:
In this case, we are explicitly asking to remove the email address... There is nothing to validate we can directly take it into account.
Alternative question would be: do we want to remove the email address? (I guess so... If we coded this, may be some customer asked it us)